charts/intel-gaudi-resource-driver/values.yaml

```
@@ -19,13 +19,26 @@ serviceAccount:
 
 kubeletPlugin:
   podAnnotations: {}
+  nodeSelector:
+    intel.feature.node.kubernetes.io/gaudi: "true"
```

This label will exist only with NFD.   I think it's better to wrap it inside `nfd.enabled` check inside deployment itself, and leave this empty, in case user wants for some reason to user his/her own label instead of NFD one.
```suggestion
    # label used when nfd.enabled is true
    #intel.feature.node.kubernetes.io/gaudi: "true"
```